src/MSBuild/XMake.cs

```diff
@@ -2914,10 +2918,12 @@ private static ILogger[] ProcessLoggingSwitches
                 verbosity = ProcessVerbositySwitch(verbositySwitchParameters[verbositySwitchParameters.Length - 1]);
             }
 
-            ArrayList loggers = ProcessLoggerSwitch(loggerSwitchParameters, verbosity);
+            ArrayList loggers = ProcessLoggerSwitch(loggerSwitchParameters, verbosity, optional: false);
```

Comment:
nit: as a bonus refactoring, move the method and its callees off of ArrayList :)